Gemfile.lock

```diff
@@ -223,7 +223,7 @@ GEM
     net-http-persistent (4.0.1)
       connection_pool (~> 2.2)
     nio4r (2.5.8)
-    nokogiri (1.13.6)
+    nokogiri (1.13.7)
```

Comment:
Are we expecting this dep., as well as others, to be updated too?

Reply:
Yep :) - bundler updates the required dependencies at that time as well